.changeset/small-books-divide.md

```diff
@@ -0,0 +1,5 @@
+---
+'@finos/legend-studio': patch
+---
+
+Add test for creating a mapping test and saving/rendering input test data and expected result data
```

Comment:
Don't think we need changelog for adding test, can we remove the content of this changeset